src/Checkout/Mixin.js

```diff
@@ -397,10 +408,13 @@ export default {
                     this.setActiveCheckout(checkoutId)
                 }
 
+                this.updateItems(checkoutId)
                 this.updateUserDetails(checkoutId)
                 this.updateBillingDetails(checkoutId)
                 this.updateShippingDetails(checkoutId)
 
+                this.activeCartCollection = this.currentCheckout
```

Comment:
Should this be a cloned checkout or do we want these to be linked / the same object? Unsure.

Reply:
You're right, this should have been cloned! Thanks!